### DISCUSSION TITLE
crop same center

### DISCUSSION THREAD

**Discussion by sharonlee12** [Project-MONAI/MONAI#7768] [General]
hello!I would like to make the same crop change to N 3D-images（RandCropByPosNegLabeld）. What should I do? should I concatenate them? On which channel is the concatenation made? Hope to receive everyone's reply！

**Comment by KumoLiu** [comment]
Hi @sharonlee12, to do a deterministic crop, using `SpatialCrop` instead of `RandCropByPosNegLabeld`.
https://github.com/Project-MONAI/MONAI/blob/ab4bd43ae4faa983d48b67e17bc0a2ddb597d038/monai/transforms/croppad/array.py#L443

And you can find data shape assumption here:
https://github.com/Project-MONAI/MONAI/wiki/MONAI-Data-Shape-Assumption

Hope it helps, thanks.
